.github/ISSUE_TEMPLATE/issue-report.yml

```diff
@@ -0,0 +1,163 @@
+    attributes:
+      label: What is the expected _correct_ behavior?
+      description: Write down the correct expected behavior that is supposed to happen after following the reproduction steps.
+      placeholder: |
+        The login disclaimer on the sign in screen should only show the configured text. The text that should be shown is: "I am a cool disclaimer!".
+    validations:
+      required: true
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version of the software
+      description: What version of OpenBullet2 are you using?
+      options:
+        - 0.3.0
+        - 0.2.5
+        - Older*
+      default: 0
+    validations:
+      required: true
+  - type: textarea
+    id: environment-information
+    attributes:
+      label: Environment
+      description: |
+        Accurately fill in as much environment details as possible. If a certain environment field is not shown in the template below, but you consider useful information, please include it.
+        Examples:
+        - **OS**: [e.g. Debian 12, Windows 11]
+        - **Linux Kernel**: [e.g. none, 5.15, 6.1, etc.]
+        - **Virtualization**: [e.g. Docker, KVM, LXC]
+        - **Browser**: [e.g. Firefox 91, Chrome 93, Safari 13, native client]
+        - **Installed Plugins**: [...]
+        - **Reverse Proxy**: [e.g. none, nginx, apache, cloudflared, etc.]
+        - **Base URL**: [e.g. none, yes: /example]
+        - **Networking**: [e.g. Host, Bridge/NAT]
+        - **Storage**: [e.g. local, NFS, cloud]
+      value: |
+        - OS:
+        - Linux Kernel:
+        - Virtualization:
+        - Browser:
+        - Plugins:
+        - Reverse Proxy:
+        - Base URL:
+        - Networking:
+        - Storage:
+      render: markdown
+    validations:
+      required: true
+  - type: markdown
+    id: general-information-logs
+    attributes:
+      value: |
+        When providing logs, please keep the following things in mind:
+        1. **DO NOT** use external paste services. If logs are too large to paste into the field, upload them as text files.
+        2. Please provide complete logs.
+          - Ensure to capture all relevant information, encompassing both the events leading up to and following the occurrence of the issue. Typically, providing 10 *lines preceding and succeeding* the problem should be adequate.
+        3. Please do not run logs through any translation program. We exclusively accept raw, untranslated logs. Particularly exercise caution if your browser automatically translates pages by default.
+          - **DO NOT** forget to censor out personal information such as public IP addresses.
+        4. Please do not include logs as screenshots, with the only exception being client logs in browsers.
+  - type: textarea
+    id: openbullet-logs
+    attributes:
+      label: OpenBullet2 logs
+      description: Please copy and paste any relevant log output.
+      render: shell
+    validations:
+      required: true
```

Comment:
I don't think this should be required, maybe there are no relevant logs because the bug doesn't cause any exceptions in the code but it's just a logical issue.